src/styles.py

```diff
@@ -279,7 +280,7 @@ def copy_style(self, template: str, new_name: str):
         style_io.write(destination, source)
 
     def complete_style(self, params):
-        for key, value in self._system_style_params.items():
+        for key, value in self.get_system_style_params().items():
```

Comment:
Any reason to change it to the getter?

Reply:
Not really, except for that I generally try to avoid accessing underscored attributes directly, especially if a getter exists. But in practice it will end up the same.

Reply:
The reason here being, that a getter will introduce (a very small) overhead. Since we are in the class, a private access would be fine imo. From outside I'm very much with you.